xgi/algorithms/simpliciality.py

```diff
@@ -0,0 +1,369 @@
+from itertools import chain, combinations
+
+import numpy as np
+from scipy.special import binom
+
+from ..core import Hypergraph
+from ..utils import Trie
+
+__all__ = [
+    "edit_simpliciality",
+    "simplicial_edit_distance",
+    "face_edit_simpliciality",
+    "mean_face_edit_distance",
+    "simplicial_fraction",
+]
+
+
+def edit_simpliciality(H, min_size=2, exclude_min_size=True):
+    """Computes the edit simpliciality.
+
+    The number of edges needed to be added
+    to a hypergraph to make it a simplicial complex.
+
+    Parameters
+    ----------
+    H : xgi.Hypergraph
+        The hypergraph of interest
+    min_size: int, default: 2
+        The minimum hyperedge size to include when
+        calculating whether a hyperedge is a simplex
+        by counting subfaces.
+    exclude_min_size : bool, optional
+        Whether to include minimal simplices when counting simplices, by default True
+
+    Returns
+    -------
+    float
+        The edit simpliciality
+
+    See Also
+    --------
+    simplicial_edit_distance
+
+    References
+    ----------
+    "The simpliciality of higher-order order networks"
+    by Nicholas Landry, Jean-Gabriel Young, and Nicole Eikmeier,
+    *EPJ Data Science* **13**, 17 (2024).
+    """
+    return 1 - simplicial_edit_distance(
+        H, min_size=min_size, exclude_min_size=exclude_min_size
+    )
+
+
+def simplicial_edit_distance(H, min_size=2, exclude_min_size=True, normalize=True):
```

Comment:
I don't think I could fully understand from the documentation alone what exactly the parameters `min_size` and `exclude_min_size` control. I had to go read the code, especially for the latter parameter. I'm not sure if there is a straightforward solution without either (a) having longer more expressive parameter names, or (b) a more thorough description in the main docstring that explains explicitly how they change the calculation, but maybe something worth thinking about a bit since I am somewhat familiar with the measures and couldn't quite understand.

Reply:
On it! Thanks for bringing this up!